packages/daemon/src/service/pipeline.ts

```diff
@@ -231,25 +245,25 @@ export class PipelineService {
       await fs.ensureDir(modelPath);
 
       // run dataCollect to download dataset.
-      await runnable.start(plugins.dataCollect.plugin, getParams(plugins.dataCollect.params, {
+      await run('dataCollect', plugins.dataCollect.plugin, getParams(plugins.dataCollect.params, {
```

Comment:
How about `run('dataCollect', ...args)`?

Reply:
What's the `args`? If you mean the args of the function `startJob`, it's not correct for function `run`.

Reply:
```js
await run('dataCollect', getParams(plugins.dataCollect.params, { ... });
```